src/textual/renderables/bar.py

```diff
@@ -4,6 +4,8 @@
 from rich.style import StyleType
 from rich.text import Text
 
+_DEFAULT_BAR_thickness = 1
```

Comment:
Much like you did for the possible colour schemes, it would be good if you had a set of all possible values somewhere so that we can do checks when needed and also a `Literal` type.

Reply:
Fixed.